packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -3,23 +3,49 @@ import { DropdownProps, DropdownState } from '../../../../components/Dropdown/Dr
 import { DropdownVariables } from './dropdownVariables'
 import { pxToRem } from '../../../../lib'
 
-const dropdownStyles: ComponentSlotStylesInput<DropdownProps & DropdownState, DropdownVariables> = {
-  root: (): ICSSInJSStyle => ({}),
+type Props = DropdownProps & DropdownState
```

Comment:
Not sure that `Props` is the best name there, `DropdownPropsAndState` may be?

Reply:
done, thx